src/lib/notify_new_path.c

```diff
@@ -21,10 +21,10 @@
 void notify_new_path(const char *path)
 {
     /* Ignore results and don't wait for response -> NULL */
-    notify_new_path_with_reponse(path, NULL);
+    notify_new_path_with_response(path, NULL);
 }
 
-int notify_new_path_with_reponse(const char *path, char **message)
+int notify_new_path_with_response(const char *path, char **message)
```

Comment:
Can you please alter https://github.com/abrt/abrt/blob/master/src/dbus/abrt_problems2_task_new_problem.c#L170 too?